cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java

```diff
@@ -92,15 +93,22 @@ public ProgramClient(ClientConfig config) {
    * @throws ProgramNotFoundException if the program with the specified name could not be found
```

Comment:
missing the debug parameter in the Javadocs. Goes above runtimeArgs; can be copied from the method after this one.